Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
```

Comment:
The "totalassets" check can be removed, we'll catch any changes to the number of files in PR review.

Reply:
Done.